doc/content/lorawan/spreading-factors/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Spreading Factors
```

Comment:
maybe add a nice picture to this chapter that shows the relationship between spreading factor, time on air, bitrate and consumption